Tests/iaas/flavor-naming/flavor_names.py

```diff
@@ -222,7 +222,9 @@ class GPU:
               "3": "Arc/Gen12.7/DG2"},
     })
     cu = OptIntAttr("#.N:SMs/A:CUs/I:EUs")
-    perf = TblAttr("Performance", {"": "Std Perf", "h": "High Perf", "hh": "Very High Perf", "hhh": "Very Very High Perf"})
+    perf = TblAttr("Performance", {"": "Std Perf", "h": "High Perf", "hh": "Very High Perf"})
+    vram = OptIntAttr("#.V:GiB VRAM")
+    vramperf = TblAttr("Performance", {"": "Std Perf", "h": "High Perf", "hh": "Very High Perf"})
```

Comment:
I think we decided to not have the "hh" indicators anymore, just a single "h" for compute power and high memory bandwidth.
Also for the vramperf, the explanation of the "h" should be "High Perf HBM memory" or similar.

Reply:
The text is saying something else. It speaks of an additional h.

Reply:
The GPU vendors may decide to create variants with extra high or low frequencies and extra high bandwidth, so let's keep the `hh` option open. I hope we don't need it, but better be safe.

Reply:
ok, I understand